SpatialGDK/Source/SpatialGDKServices/Private/SpatialGDKServicesModule.cpp

```diff
@@ -20,8 +20,14 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKServices);
 
 IMPLEMENT_MODULE(FSpatialGDKServicesModule, SpatialGDKServices);
 
+#if PLATFORM_WINDOWS
 const FString SpatialExe = TEXT("spatial.exe");
 const FString SpotExe = FSpatialGDKServicesModule::GetSpatialGDKPluginDirectory(TEXT("SpatialGDK/Binaries/ThirdParty/Improbable/Programs/spot.exe"));
+#elif PLATFORM_MAC
+const FString SpatialExe = TEXT("/usr/local/bin/spatial");
```

Comment:
Can't this just be `spatial`?

Reply:
unfortunately not, because you lose most environment variables and the PATH also looks different when starting GI applications (you'll only have `/usr/bin` in your path which is a bit useless). Played around with it quite a bit yesterday and I think the safest solution is to say that we will be explicitly looking in there and then either tell them in our setup guide that this is the place or add a field somewhere in the settings where they can optionally set a new path to spatial.

Reply:
Would it worth appending /usr/local/bin to path variable then as a part of setup process? Or creating an alias for /usr/local/bin/spatial?

Reply:
simply appending the PATH will have no affect to the PATH that will be used by Unreal. :( I tried loads of different ways, but the only way to get it working is by doing this inside the Xcode project. However, as soon as you create a new project or regenerate the file, that path will be removed and it needs to be re-done.

I think there are two potential solutions that could work fairly well:
1. Have this path hardcoded and add an additional step to symlink spatial to /usr/local/bin, if it spatial doesn't exist there. This could even be added to our Setup.sh
2 Have a new field in the Unreal settings that people can update to the right path. This will cause errors in the editor until it's fixed, but at least it's visible.

Reply:
I see, yes, it's better to not complicate the XCode setup. 

I think it's okay to hardcode the path (or better add it to settings config file with a default `/usr/local/bin` value), then have a jira feature request, to expose the path via Unreal Settings UI. 

I'd say it's up to users to decide afterwards whether they want symlinks.